source/gameengine/VideoTexture/ImageRender.cpp

```diff
@@ -888,6 +889,7 @@ ImageRender::ImageRender (KX_Scene *scene, KX_GameObject *observer, KX_GameObjec
     m_mirror(mirror),
     m_clip(100.f)
 {
+	m_type = IMAGE_MIRROR;
```

Comment:
@lordloki : Hi, It is in public. If I put it under protected, I have the same issue. Maybe I'll try to change that later.